examples/init_api_client.py

```diff
@@ -37,13 +76,15 @@ def parse_arguments():
                         default='admin', help="password for authentication")
     parser.add_argument("--domain_type", action="store",
                         default='LOCAL', help="domain type for authentication")
-    parser.add_argument("--data_sources_csv", action="store",
-                        default='data_sources.csv', help="Name of CSV file")
-    args = parser.parse_args()
-    return args
+
+    # Network Insight as a service (NIAAS) parameters
+    parser.add_argument('--refresh_token', action='store',
```

Comment:
We need in detail steps to get refresh_token for NIaaS setups

Reply:
Updated.